### PR TITLE
fix: fix update-issue-sprint.yml permissions and support QA Sprint update

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue #

### CODE REVIEW

File: .github/workflows/update-issue-sprint.yml [hunk 1]
```diff
@@ -34,9 +39,45 @@ jobs:
...
+      uses: derekbit/move-to-next-iteration@master
+      if: steps.check_sprint_build_required.outcome == 'success'
+      with:
+        owner: longhorn
+        number: 4
+        token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
+        iteration-field: Sprint
+        iteration: current
+        new-iteration: none
+        excluded-statuses: "Review,Testing,Closed"
```
Comment (innobead) [inline .github/workflows/update-issue-sprint.yml:66]: What's the purpose of `testing` status in the QA sprint?

**Review by innobead — COMMENTED** [review]
LGTM, just one question.

File: .github/workflows/update-issue-sprint.yml (hunk 1 above)
Comment (derekbit) [reply .github/workflows/update-issue-sprint.yml:66]: > What's the purpose of testing status in the QA sprint?

After merging the PR and before closing it, the test case should be re-run to make sure nothing broken.
WDYT?

Comment (innobead) [reply .github/workflows/update-issue-sprint.yml:66]: Sounds good. cc @longhorn/qa

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
cc @Yu-Jack @bk201

**Pull request merged**